src/backend/users/views.py

```diff
@@ -183,3 +198,19 @@ def put(self, request, *args, **kwargs):
         serializer = UserSerializer(User.objects.get(username=request.user))
         content = self.repackage_for_frontend(serializer.data)
         return Response(content)
+
+
+class SignUpPage(APIView):
+    def post(self, request, **kwargs):
+        signup_message = "Thank you! You'll receive an email shortly with a link to complete your account. Click the link to create your profile and get started with ECHO."
```

Comment:
This can create challenges for translation because these are user-visible strings that don't pass through the frontend translation framework. I wouldn't change anything now in the interest of time, but for the future, if there's translation happening client-side, I'd have the backend send back message keys, maybe something like `'SignIn.AccountExists'` or `'SignIn.InvalidEmail'`, and then use those to determine what to display to the user client-side.